media/css/firefox/personal.less

```diff
@@ -0,0 +1,194 @@
+@import "../sandstone/lib.less";
+
+@font-face {
+    font-family: 'FiraSans Regular';
+    src: url('/media/fonts/FiraSans-Regular.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'FiraSans Bold';
+    src: url('/media/fonts/FiraSans-Bold.woff') format('woff');
+    font-weight: normal;
```

Comment:
Instead separate families for Regular and Bold, we can just have "Fira Sans" as the family name and use <code>font-weight: bold;</code> same as we've done for Open Sans (see https://github.com/mozilla/bedrock/blob/master/media/css/sandstone/fonts.less). 

Fira Sans Light does need to be its own family name, though, since browsers have poor support for incremental weights (200, 500, 700, etc).